controller/src/enhancements/player.rs

```diff
@@ -145,7 +145,7 @@ impl PlayerESP {
                 .context("invalid player name")?
                 .to_string()
         } else {
-            "unknown".to_string()
+            return Ok(None);
```

Comment:
Mabe add a warning message here, indicating that handle xyz / a player pawn at address xyz does not contain a valid controller? I'm wondering under which circumstances this will happen.